compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/HttpFiltersDesugar.java

```diff
@@ -175,7 +146,13 @@ private boolean isHttpResource(BLangFunction resourceNode) {
                 CALLER_TYPE_NAME.equals(tsymbol.name.value);
     }
 
-    private void addFilterStatements(BLangFunction resourceNode, SymbolEnv env) {
+    /**
+     * Apply filter statements.
+     *
+     * @param resourceNode The resource to apply filter on if it is http
```

Comment:
The "if it is http" part is no longer relevant, right? Since we do the check before calling this method now?

```suggestion
     * @param resourceNode The resource to apply filters on
```